lib/rex/post/meterpreter/ui/console/command_dispatcher/priv/elevate.rb

```diff
@@ -46,6 +49,25 @@ def name
 
 
   #
+  # Returns the description of the technique(s)
+  #
+  def translate_technique_index(index)
+    translation = ''
+
+    case index
+    when 0
+      desc = ELEVATE_TECHNIQUE_DESCRIPTION.dup
+      desc.shift
+      translation = desc
```

Comment:
``` ruby
translation = ELEVATE_TECHNIQUE_DESCRIPTION[1, ELEVATE_TECHNIQUE_DESCRIPTION.length]
```
:) I'm code-golfing where it really isn't necessary! Anyone would think I'm trying to avoid working with Squid :-/